bigquery/cloud-client/test/config.js

```diff
@@ -0,0 +1,255 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {BigQuery} = require('@google-cloud/bigquery');
+const crypto = require('crypto');
+
+// Define enum for HTTP codes
+const HTTP_STATUS = {
+  NOT_FOUND: 404,
+};
+
+// Generate a unique prefix using a random UUID to ensure uniqueness across test runs
+function createRandomPrefix() {
+  return `nodejs_test_${crypto.randomUUID().replace(/-/g, '').substring(0, 8)}`;
+}
+
+const PREFIX = createRandomPrefix();
+console.log(`Generated test prefix: ${PREFIX}`);
+
+const ENTITY_ID = 'cloud-developer-relations@google.com'; // Group account
```

Comment:
question: Do we know that using this entity ID "works"? I think we should look at the IAM samples to see if there is a better pattern used there.